src/inspector_agent.cc

```diff
@@ -89,18 +90,18 @@ void SendVersionResponse(inspector_socket_t* socket) {
   SendHttpResponse(socket, buffer, len);
 }
 
-void SendTargentsListResponse(inspector_socket_t* socket) {
+void SendTargentsListResponse(inspector_socket_t* socket, int port) {
   const char LIST_RESPONSE_TEMPLATE[] =
       "[ {"
       "  \"description\": \"node.js instance\","
       "  \"devtoolsFrontendUrl\": "
             "\"https://chrome-devtools-frontend.appspot.com/serve_file/"
-            "@4604d24a75168768584760ba56d175507941852f/inspector.html\","
+            "@%s/inspector.html?experiments=true&v8only=true&ws=localhost:%d%s\","
       "  \"faviconUrl\": \"https://nodejs.org/static/favicon.ico\","
       "  \"id\": \"%d\","
       "  \"title\": \"%s\","
       "  \"type\": \"node\","
-      "  \"webSocketDebuggerUrl\": \"ws://%s\""
+      "  \"webSocketDebuggerUrl\": \"ws://localhost:%d%s\""
```

Comment:
@auchenberg I'll dig into this some more and maybe make a different PR that builds on this one.